Login/login.css

```diff
@@ -0,0 +1,35 @@
+.login {
+  /* padding: 0 70px; */
+}
+
+.login-heading {
+  text-align: center;
+  padding-bottom: 40px;
+}
+
+.login label {
+  margin: 0;
+}
+
+.login .input-group input {
+  width: 375px;
+}
+
+.div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
Try to give better names for the classes & for units rems can be used instead of pixels. Rest looks good to me.